Error due to preact-markup when linking `master` in downstream project

When I link the latest changes locally to the Camunda Modeler, I got the following error

```error
ERROR in /Users/niklas.kiefer/bpmn-io/bpmn-js-properties-panel/node_modules/preact-markup/dist/preact-markup.module.js
Module not found: Error: Can't resolve 'preact' in '/Users/niklas.kiefer/bpmn-io/bpmn-js-properties-panel/node_modules/preact-markup/dist'
 @ /Users/niklas.kiefer/bpmn-io/bpmn-js-properties-panel/node_modules/preact-markup/dist/preact-markup.module.js 1:0-42 1:2099-2100 1:3759-3760
 @ /Users/niklas.kiefer/bpmn-io/bpmn-js-properties-panel/dist/index.esm.js
 @ /Users/niklas.kiefer/bpmn-io/camunda-bpmn-js/lib/camunda-platform/Modeler.js
 @ ./src/app/tabs/bpmn/modeler/BpmnModeler.js
 @ ./src/app/tabs/bpmn/modeler/index.js
 @ ./src/app/tabs/bpmn/BpmnEditor.js
 @ ./src/app/tabs/bpmn/BpmnTab.js
 @ ./src/app/tabs/bpmn/index.js
 @ ./src/app/TabsProvider.js
 @ ./src/app/index.js
 @ ./src/index.js
 @ multi ./src/index.js
 ```
 

_Originally posted by @pinussilvestrus in https://github.com/bpmn-io/bpmn-js-properties-panel/issues/549#issuecomment-1017570113_

--- a/rollup.config.js
+++ b/rollup.config.js
@@ -9,6 +9,7 @@ import resolve from '@rollup/plugin-node-resolve';
 import pkg from './package.json';
 
 const nonbundledDependencies = Object.keys({ ...pkg.dependencies, ...pkg.peerDependencies });
+const nonExternalDependencies = [ 'preact-markup' ];
 
 export default [
   {
@@ -25,11 +26,12 @@ export default [
         file: pkg.module
       }
     ],
-    external: id => nonbundledDependencies.find(dep => id.startsWith(dep)),
+    external: externalDependencies(),
     plugins: [
       alias({
         entries: [
-          { find: 'react', replacement: '@bpmn-io/properties-panel/preact/compat' }
+          { find: 'react', replacement: '@bpmn-io/properties-panel/preact/compat' },
+          { find: 'preact', replacement: '@bpmn-io/properties-panel/preact' }
         ]
       }),
       reactSvg(),
@@ -54,3 +56,11 @@ export default [
     ]
   }
 ];
+
+
+function externalDependencies() {
+  return id => {
+    return nonbundledDependencies.find(dep => id.startsWith(dep)) &&
+      !nonExternalDependencies.find(dep => id.startsWith(dep));
+  };
+}

